src/components/options/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { h, Component } from 'preact';
+import * as style from './style.scss';
+import { bind } from '../../lib/util';
+import { ImageType, CodecOptions } from '../app';
+
+const CodecConfigurations: {
+  [type: string]: (props: ChildProps) => JSX.Element | null
+} = {
+  original: () => null
+};
+
+type ChildProps = {
+  options: CodecOptions,
+  updateOption(e: Event): void,
+  setOption(key: string, value: any): void
+};
+
+CodecConfigurations.jpeg = ({ options, updateOption }: ChildProps) => (
```

Comment:
Yeah, would be nicer to move this to a file next to the codec wrapper (or even into the codec wrapper file? Your call).

Also I think we should call the codec for what it is: MozJpeg

Reply:
Agree on both counts.

Reply:
yup, I'll collocate these. I'm going to hoist that directory though.